src/windows/enable-nestedhyperv.ps1

```diff
@@ -0,0 +1,106 @@
+$scriptStartTime = get-date -f yyyyMMddHHmmss
```

Comment:
Can you read through https://github.com/Azure/repair-script-library/blob/master/doc/adding_new_scripts.md 

and use the existing Logging functions to log messages and return the right status on failure and success?